src/Components/NotificationIntegrationModal/Components/NotificationIntegrationModal.jsx

```
@@ -19,335 +19,406 @@ import useNotifications from "../Hooks/useNotification";
-  setMonitor(prev => ({
-    ...prev,
-    notifications: filteredNotifications
-  }));
-  
-  onClose();
-};
+	const { t } = useTranslation();
+	const theme = useTheme();
+	const [tabValue, setTabValue] = useState(0);
+
+	const [loading, _, sendTestNotification] = useNotifications();
```

ESLint

[error] '_' is assigned a value but never used.

(no-unused-vars)